CMakeLists.txt

```diff
@@ -1,5 +1,17 @@
 cmake_minimum_required(VERSION 3.10)
 
-file(GLOB_RECURSE SOURCES "src/*.c" "include/*.h")
-add_library(mog STATIC ${SOURCES})
+include(GNUInstallDirs)
+
+file(GLOB_RECURSE SOURCES "src/*.c")
+file(GLOB_RECURSE HEADERS "include/*.h")
+
+add_library(mog STATIC ${HEADERS} ${SOURCES})
 target_include_directories(mog PUBLIC include)
+set_target_properties(mog PROPERTIES PUBLIC_HEADER ${HEADERS})
+
+install(TARGETS mog
+  RUNTIME DESTINATION "bin"
```

Comment:
Why not `CMAKE_INSTALL_BINDIR` to be consistent?

Otherwise looks fine :100:

Reply:
Done! Thank you for the suggestion, I totally forgot about it.